openvdb/openvdb/tree/ValueAccessor.h

```diff
@@ -2454,8 +3018,8 @@ class ValueAccessor3 : public ValueAccessorBase<_TreeType, IsSafe>
             }
             return BaseT::mTree->root().template probeNodeAndCache<NodeT>(xyz, *this);
         }
-        return nullptr;
-        OPENVDB_NO_UNREACHABLE_CODE_WARNING_END
+        // No point trying to cache as this NodeT isn't part of this accessors chain
+        return BaseT::mTree->root().template probeNode<NodeT>(xyz);
```

Comment:
Bugfix - even if `NodeT` isn't being cached it might exist and a nullptr will always be returned